assets/css/dashboard/new-pa-message/selected-stations-summary.scss

```diff
@@ -0,0 +1,80 @@
+.selected-stations-summary {
+  display: flex;
+  align-items: center;
+  height: 52px;
+  border-radius: 8px;
+  background-color: $cool-gray-30;
+  margin: 40px 40px 0 40px;
+  padding: 15px 18px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 30px;
+
+  .label {
+    margin-right: 8px;
+  }
+
+  span {
+    color: $text-secondary;
+  }
+
+  .bi {
```

Comment:
It's actually a bootstrap style automatically added to icons in the `react-bootstrap-icons` package. This is a good call out because there's no reason I can't also add a custom class so we can track them down more easily.